Added a Pull Request Template

KushalXCoder opened the pull request:
Based on the issue description and some online learning, I have added things that I thought would be required. But, still, review it and suggest any additions or deletions in it.

KushalXCoder committed:
Added a Pull Request Template

Saumya40-codes commented on .github/ISSUE_TEMPLATE/bug_report.md:
Please keep both bug_report and feature_request template as it is. No need to delete

KushalXCoder replied:
Added again.

Saumya40-codes commented:
LGTM overall. Thanks

KushalXCoder committed:
Added the bug_request and feature_request

KushalXCoder committed:
Pushed bug_report and feature_request into ISSUE_TEMPLATE

Saumya40-codes commented:
Please don't combine 2 issue fixes in one PR, might get hard to keep track going forward

Saumya40-codes commented:
Just a suggestion
### This is irrelevant to this issue but might create problem with your pr going foward

You can go to your cloned/forked repo and click `Sync fork` option or `update branch` option  (make sure current branch is `main` there)

then 
```
 git checkout main
```

and 

```
git pull
```

Then you can create another branch to work on. This make sure you are working on latest codebase, might decrease the chance of merge conflict or any other deps issue

KushalXCoder commented:
I didn't merge two issues, I just added back the bug_report and feature_request which I deleted earlier, and added them in the original folder in which they were, which I deleted at starting.

Saumya40-codes commented:
> I didn't merge two issues, I just added back the bug_report and feature_request which I deleted earlier, and added them in the original folder in which they were, which I deleted at starting.

imo this is part of the new issue which you were working on #14 
![image](https://github.com/user-attachments/assets/1fc3ac9a-1e8c-453c-8bba-eb7737abc659)

Saumya40-codes commented:
> imo this is part of the new issue which you were working on

We can go on with this, but make sure to make changes of different issue not on the same branch :)

KushalXCoder commented:
While committing the last time, I, by mistake, did git add . instead of a specific file, and hence, two issues got pushed. But I will take care of it the next time.
But thanks for accepting the PR :)
